Tasks/zip.js

```diff
@@ -1,18 +1,15 @@
 // Zip two arrays, [1, 2] and [3, 4] -> [[1, 3], [2, 4]]
+'use strict';
 
-const zip = function (a = [], b = []) {
-  let i = 0;
-  j = 0;
-  for (x of b) {
-    CELL = [a[i++], x];
-    if (i < j) {
-      delete a[i++];
-    } else {
-      (() => (b[j++] = CELL))();
-    }
-    if (CELL[0] == undefined) b.length -= 1
+const zip = (arr1, arr2) => {
+  let i = 0, 
+      j = 0;
+  for (const element of arr2) {
+    const tempCell = [arr1[i++], element];
+    i < j ? delete arr1[i++] : arr2[j++] = tempCell;
+    if (tempCell[0] == null) arr2.length -= 1;
```

Comment:
- Try not to change incoming parameters 
- Add the third "=": `if (tempCell[0] === null) arr2.length -= 1;`

Reply:
Thank you! Now the arguments don't change and code works without conditions